diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/components/Reflection.js b/node_modules/docusaurus-plugin-typedoc-api/lib/components/Reflection.js
index 5b55140..a356497 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/components/Reflection.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/components/Reflection.js
@@ -11,6 +11,7 @@ const Members = require('./Members.js');
 const MemberSignatures = require('./MemberSignatures.js');
 const Parameter = require('./Parameter.js');
 const Type = require('./Type.js');
+const SourceLink = require('./SourceLink.js');
 const TypeParameters = require('./TypeParameters.js');
 const jsxRuntime = require('react/jsx-runtime');
 
@@ -87,7 +88,10 @@ function Reflection({
       className: "tsd-panel",
       children: [/*#__PURE__*/jsxRuntime.jsx("h3", {
         className: "tsd-panel-header tsd-before-signature",
-        children: "Callable"
+        children: [
+          "Callable",
+          /*#__PURE__*/jsxRuntime.jsx(SourceLink.SourceLink, { sources: reflection.sources })
+        ]
       }), /*#__PURE__*/jsxRuntime.jsx("div", {
         className: "tsd-panel-content",
         children: /*#__PURE__*/jsxRuntime.jsx(MemberSignatures.MemberSignatures, {
diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/components/SourceLink.js b/node_modules/docusaurus-plugin-typedoc-api/lib/components/SourceLink.js
index 9de02ac..96491ef 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/components/SourceLink.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/components/SourceLink.js
@@ -1,21 +1,13 @@
 'use strict';
 
-const useDocusaurusContext = require('@docusaurus/useDocusaurusContext');
 const useGitRefName = require('../hooks/useGitRefName.js');
 const jsxRuntime = require('react/jsx-runtime');
-const _interopDefault = e => e && e.__esModule ? e : {
-  default: e
-};
-const useDocusaurusContext__default = /*#__PURE__*/_interopDefault(useDocusaurusContext);
 function replaceWithSrc(url) {
-  // Always link the source file
-  return url.replace(/\/(dts|dist|lib|build|es|esm|cjs|mjs)\//, '/src/');
+  return url.replace('bc-stubs/bc', 'bc');
 }
 function SourceLink({
   sources = []
 }) {
-  const _useDocusaurusContext = useDocusaurusContext__default.default(),
-    siteConfig = _useDocusaurusContext.siteConfig;
   const gitRefName = useGitRefName.useGitRefName();
   if (sources.length === 0) {
     return null;
@@ -23,7 +15,7 @@ function SourceLink({
   return /*#__PURE__*/jsxRuntime.jsx(jsxRuntime.Fragment, {
     children: sources.map(source => /*#__PURE__*/jsxRuntime.jsx("a", {
       className: "tsd-anchor",
-      href: source.url || `https://${siteConfig.githubHost}${siteConfig.githubPort ? `:${siteConfig.githubPort}` : ''}/${siteConfig.organizationName}/${siteConfig.projectName}/blob/${gitRefName}/${replaceWithSrc(source.fileName)}#L${source.line}`,
+      href: source.url || `https://gitgud.io/bananarama92/Bondage-College/-/blob/${gitRefName}/BondageClub/${replaceWithSrc(source.fileName)}#L${source.line}`,
       rel: "noreferrer",
       target: "_blank",
       children: /*#__PURE__*/jsxRuntime.jsx("i", {
diff --git a/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/data.js b/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/data.js
index 56ddf05..bd32d7b 100644
--- a/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/data.js
+++ b/node_modules/docusaurus-plugin-typedoc-api/lib/plugin/data.js
@@ -76,7 +76,7 @@ async function generateJson(projectRoot, entryPoints, outFile, options) {
     skipErrorChecking: true,
     // stripYamlFrontmatter: true,
     // Only emit when using project references
-    emit: shouldEmit(projectRoot, tsconfig),
+    emit: 'none',
     // Only document the public API by default
     excludeExternals: true,
     excludeInternal: true,
