Index: /docusaurus-plugin-typedoc-api/lib/components/Reflection.js
===================================================================
--- /docusaurus-plugin-typedoc-api/lib/components/Reflection.js
+++ /docusaurus-plugin-typedoc-api/lib/components/Reflection.js
@@ -10,8 +10,9 @@
 const Members = require('./Members.js');
 const MemberSignatures = require('./MemberSignatures.js');
 const Parameter = require('./Parameter.js');
 const Type = require('./Type.js');
+const SourceLink = require('./SourceLink.js');
 const TypeParameters = require('./TypeParameters.js');
 const jsxRuntime = require('react/jsx-runtime');
 
 // https://github.com/TypeStrong/typedoc-default-themes/blob/master/src/default/templates/reflection.hbs
@@ -86,9 +87,12 @@
     }), 'signatures' in reflection && reflection.signatures && reflection.signatures.length > 0 && /*#__PURE__*/jsxRuntime.jsxs("section", {
       className: "tsd-panel",
       children: [/*#__PURE__*/jsxRuntime.jsx("h3", {
         className: "tsd-panel-header tsd-before-signature",
-        children: "Callable"
+        children: [
+          "Callable",
+          /*#__PURE__*/jsxRuntime.jsx(SourceLink.SourceLink, { sources: reflection.sources })
+        ]
       }), /*#__PURE__*/jsxRuntime.jsx("div", {
         className: "tsd-panel-content",
         children: /*#__PURE__*/jsxRuntime.jsx(MemberSignatures.MemberSignatures, {
           sigs: reflection.signatures
Index: /docusaurus-plugin-typedoc-api/lib/components/SourceLink.js
===================================================================
--- /docusaurus-plugin-typedoc-api/lib/components/SourceLink.js
+++ /docusaurus-plugin-typedoc-api/lib/components/SourceLink.js
@@ -1,30 +1,22 @@
 'use strict';
 
-const useDocusaurusContext = require('@docusaurus/useDocusaurusContext');
 const useGitRefName = require('../hooks/useGitRefName.js');
 const jsxRuntime = require('react/jsx-runtime');
-const _interopDefault = e => e && e.__esModule ? e : {
-  default: e
-};
-const useDocusaurusContext__default = /*#__PURE__*/_interopDefault(useDocusaurusContext);
 function replaceWithSrc(url) {
-  // Always link the source file
-  return url.replace(/\/(dts|dist|lib|build|es|esm|cjs|mjs)\//, '/src/');
+  return url.replace('bc-stubs/bc', 'bc');
 }
 function SourceLink({
   sources = []
 }) {
-  const _useDocusaurusContext = useDocusaurusContext__default.default(),
-    siteConfig = _useDocusaurusContext.siteConfig;
   const gitRefName = useGitRefName.useGitRefName();
   if (sources.length === 0) {
     return null;
   }
   return /*#__PURE__*/jsxRuntime.jsx(jsxRuntime.Fragment, {
     children: sources.map(source => /*#__PURE__*/jsxRuntime.jsx("a", {
       className: "tsd-anchor",
-      href: source.url || `https://${siteConfig.githubHost}${siteConfig.githubPort ? `:${siteConfig.githubPort}` : ''}/${siteConfig.organizationName}/${siteConfig.projectName}/blob/${gitRefName}/${replaceWithSrc(source.fileName)}#L${source.line}`,
+      href: source.url || `https://gitgud.io/silizia/Bondage-College/-/blob/${gitRefName}/BondageClub/${replaceWithSrc(source.fileName)}#L${source.line}`,
       rel: "noreferrer",
       target: "_blank",
       children: /*#__PURE__*/jsxRuntime.jsx("i", {
         className: "codicon codicon-file-code"
Index: /docusaurus-plugin-typedoc-api/lib/plugin/data.js
===================================================================
--- /docusaurus-plugin-typedoc-api/lib/plugin/data.js
+++ /docusaurus-plugin-typedoc-api/lib/plugin/data.js
@@ -75,9 +75,9 @@
     includeVersion: true,
     skipErrorChecking: true,
     // stripYamlFrontmatter: true,
     // Only emit when using project references
-    emit: shouldEmit(projectRoot, tsconfig),
+    emit: 'none',
     // Only document the public API by default
     excludeExternals: true,
     excludeInternal: true,
     excludePrivate: true,
